### PR TITLE
New: Add autofixer to `require-meta-has-suggestions` rule

### PR DESCRIPTION
**Pull request by bmish**
If suggestions are detected, add `hasSuggestions: true`.

### CODE REVIEW

File: docs/rules/require-meta-has-suggestions.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Require suggestable rules to implement a `meta.hasSuggestions` property (require-meta-has-suggestions)
 
+⚒️ The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#-fix) can automatically fix some of the problems reported by this rule.
```
Comment (aladdin-add) [inline docs/rules/require-meta-has-suggestions.md:3]:
```suggestion
⚒️ The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#--fix) can automatically fix some of the problems reported by this rule.
```

**Review by aladdin-add — APPROVED** [review]
LGTM, thanks!

File: docs/rules/require-meta-has-suggestions.md (hunk 1 above)
Comment (bmish) [reply docs/rules/require-meta-has-suggestions.md]: Need to fix the URL for all rule docs and the test: https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/pull/169

Comment (aladdin-add) [reply docs/rules/require-meta-has-suggestions.md]: 👍

**Comment by aladdin-add** [comment]
https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/pull/169 has been merged. can you rebase the master please?
thanks!

**Comment by bmish** [comment]
Fixed.

**Pull request merged**